requirements/tests.txt

```diff
@@ -0,0 +1,7 @@
+Flask
```

Comment:
Flask is a requirement for the application under test. All other requirements are for the tests themselves. I would create a `base.txt` for the application requirements, and make `tests.txt` include these.